tests/test_utils.py

```diff
@@ -1,36 +1,210 @@
+    assert phrosty.utils.get_transient_mjd(trns) == pytest.approx( endpoints, abs=0.1 )
 
 def test_get_transient_zcmb():
     oid = 20172782
     zcmb = np.float32(0.3601)
 
     tzcmb = get_transient_zcmb(oid)
 
-    assert zcmb == tzcmb
+    assert zcmb == pytest.approx( tzcmb, rel=1e-5 )
 
 def test_get_transient_peakmjd():
     oid = 20172782
     mjd = np.float32(62476.507812)
 
     tmjd = get_transient_peakmjd(oid)
 
-    assert mjd == tmjd
\ No newline at end of file
+    assert mjd == pytest.approx( tmjd, abs=0.001 )
+
+
+def test_get_transient_info():
+    oid = 20172782
+    coords = (7.551093401915147, -44.80718106491529, 62450.0, 62881.0)
+    assert phrosty.utils.get_transient_info( oid ) == pytest.approx( coords, rel=1e-6 )
+
+
+def test_transient_in_or_out():
+    oid = 20172782
+    mjd0 = 62450.0
+    mjd1 = 62881.0
+    band = 'R062'
+
+    in_tab, out_tab = phrosty.utils.transient_in_or_out( oid, mjd0, mjd1, band )
+    assert len(in_tab) == 53
+    assert len(out_tab) == 82
+    assert set(in_tab.columns) == { 'filter', 'pointing', 'sca' }
+    assert set(out_tab.columns) == { 'filter', 'pointing', 'sca' }
+
+@pytest.mark.skip( reason="Not used in pipeline.py, test is a TODO" )
+def test_get_templates():
+    assert False
+
+@pytest.mark.skip( reason="Not used in pipeline.py, test is a TODO" )
+def test_get_science():
+    assert False
+
+def test_get_mjd_limits():
+    start, end = phrosty.utils.get_mjd_limits()
+    assert ( start, end ) == pytest.approx( (62000.02139, 63563.0579), abs=0.1 )
+
+def test_get_radec_limits():
+    val = phrosty.utils.get_radec_limits()
+    assert val['ra'] == pytest.approx( [6.97879, 12.0204], abs=0.001 )
+    assert val['dec'] == pytest.approx( [-46.5199, -41.4786], abs=0.001 )
+
+
+def test_get_mjd():
+    pointing = 0
+    test_path = os.path.join(os.path.dirname(__file__), 'testdata', 'Roman_TDS_obseq_11_6_23.fits')
+    with fits.open(test_path) as tp:
+        tobseq = Table(tp[1].data)
+    tmjd = float(tobseq['date'][int(pointing)])
+    mjd = get_mjd(pointing)
+
+    assert tmjd == mjd
+
+
+def test_pointings_near_mjd():
+    obseq = Table.read( phrosty.utils.obseq_path )
+
+    ptgs = phrosty.utils.pointings_near_mjd( 62000.04011 )
+    assert len(ptgs) == 383
+    assert np.all( obseq[ptgs]['date'] > 62000.04011 - 3 )
+    assert np.all( obseq[ptgs]['date'] < 62000.04011 + 3 )
+    ptgs = phrosty.utils.pointings_near_mjd( 62000.04011, window=1 )
+    assert len(ptgs) == 220
+    assert np.all( obseq[ptgs]['date'] > 62000.04011 - 1 )
+    assert np.all( obseq[ptgs]['date'] < 62000.04011 + 1 )
+
+    ptgs = phrosty.utils.pointings_near_mjd( 10000 )
+    assert len(ptgs) == 0
+
+def test_get_mjd_info():
+    mjd0 = 62000.04011 - 3
+    mjd1 = 62000.04011 + 3
+
+    info = phrosty.utils.get_mjd_info( mjd0, mjd1 )
+    assert len(info) == 383
+    assert set( info['filter'] ) == { 'H158', 'F184', 'R062', 'K213', 'Y106', 'Z087', 'J129' }
+
+
+def test_get_exptime():
```

Comment:
Extra points if you can give a reference for why these are the expected exposure times.  I mention this in particular because this test fill fail once we start running on real data or even sims of different exposure times and some poor grad student is going to be assigned to track this down.